### PR TITLE
ODK base release product

### PR DESCRIPTION
**Pull request by matentzn**
(requires robot alpha because of remove command)

This requests needs a more thorough look at to be sure it is according to the new @balhoff specs

### CODE REVIEW

File: template/src/ontology/Makefile [hunk 1]
```diff
@@ -49,7 +49,11 @@ $(ONT).owl: $(SRC)
 	$(ROBOT)  reason -i $< -r ELK -e none relax reduce -r ELK annotate -V $(BASE)/releases/`date +%Y-%m-%d`/$(ONT).owl -o $@
 $(ONT).obo: $(ONT).owl
 	$(ROBOT) convert --check false -i $< -f obo -o $(ONT).obo.tmp && mv $(ONT).obo.tmp $@
-
+	
+$(ONT)-base.owl: $(SRC) ../patterns/definitions.owl
+	$(ROBOT) remove --input $< --select imports --output $@ &&\
+	$(ROBOT) merge -i $@ -i ../patterns/definitions.owl -o $@ &&\
```
Comment (balhoff) [inline template/src/ontology/Makefile]: @matentzn I think you should add an `annotate` here for the ont IRI and version IRI.

Comment (matentzn) [reply template/src/ontology/Makefile]: Like:
```
annotate -V $(BASE)/releases/`date +%Y-%m-%d`/$(ONT)-base.owl
```

and

```
annotate --ontology-iri $(BASE)/$(ONT)-base.owl
```

Comment (balhoff) [reply template/src/ontology/Makefile]: Yep

**Comment by matentzn** [comment]
@balhoff hope that works. 

Build fails because of outdated OSKFULL version on dockerhub. Will fix this tomorrow! So if you are down with the changes proposed here, I will merge them tomorrow when the new ODK is on dockerhub.

File: template/src/ontology/Makefile [hunk 2]
```diff
@@ -49,7 +49,11 @@ $(ONT).owl: $(SRC)
 	$(ROBOT)  reason -i $< -r ELK -e none relax reduce -r ELK annotate -V $(BASE)/releases/`date +%Y-%m-%d`/$(ONT).owl -o $@
 $(ONT).obo: $(ONT).owl
 	$(ROBOT) convert --check false -i $< -f obo -o $(ONT).obo.tmp && mv $(ONT).obo.tmp $@
-
+	
+$(ONT)-base.owl: $(SRC) ../patterns/definitions.owl
+	$(ROBOT) remove --input $< --select imports --output $@ &&\
+	$(ROBOT) merge -i $@ -i ../patterns/definitions.owl annotate -V $(BASE)/releases/`date +%Y-%m-%d`/$(ONT)-base.owl annotate --ontology-iri $(BASE)/$(ONT)-base.owl -o $@ &&\
```
Comment (balhoff) [inline template/src/ontology/Makefile]: I think you can do this in one `annotate`, but doesn't matter much.

**Pull request merged**